cgp/hl_api.py

```diff
@@ -59,6 +64,9 @@ def evolve(
     # Main loop: -1 offset since the last loop iteration will still increase generation by one
     while pop.generation < max_generations - 1:
 
+        if reorder_genome:
+            pop.reorder_genome()
```

Comment:
i feel like this call should be done from `MuPlusLambda.step` right before creating a new offspring generation. seems more natural to me than here.

Reply:
I agree, moved it there